cassandra-3/src/main/java/io/debezium/connector/cassandra/Cassandra3CommitLogReadHandlerImpl.java

```diff
@@ -109,7 +120,9 @@ enum PartitionType {
          */
         COUNTER;
 
-        static final Set<PartitionType> supportedPartitionTypes = new HashSet<>(Arrays.asList(PARTITION_KEY_ROW_DELETION, ROW_LEVEL_MODIFICATION));
```

Comment:
@vjuranek in the future we way think about extracting this stuff to some common helper class, same with supportedPartitionTypes and related things. I just left it here as is now, we might realise that some operations are doable in Cassandra 4 only (or vice versa) so having these variables "global" is not a good idea _yet_.